osier/models/dispatch.py

```diff
@@ -134,8 +145,7 @@ class DispatchModel():
     penalty : float
         The penalty applied to the objective function to eliminate
         simultaneous charging and discharging. Users may need to tune
-        this parameter. Default is 1.0.
-
+        this parameter. Default is 1e-4.
```

Comment:
What is the justification for changing this default value? It looks like the value was already at 1e-4, so is this just an update to the documentation?

Reply:
Yup, the documentation was wrong before.